packages/simulator/utils.js

```diff
@@ -15,7 +15,7 @@ export async function injectPanel(containerNode) {
     // this contains all sorts of CSS tags etc
     containerNode.innerHTML = rawPanelHtml
     // keep only the Alpine components in the panel
-    const panelAppHtml = Array.from(containerNode.querySelectorAll('[x-data]'))
+    const panelAppHtml = Array.from(containerNode.querySelectorAll('#devtools-container > [x-data]'))
```

Comment:
Thinking this will only work on the simulator. Will see what i can do. It's otherwise duplicating the settings page.

Reply:
How about adding `data-testid`s to the panel "root" components, `data-testid="panel-root"` maybe?

(would have to be added to both https://github.com/alpine-collective/alpinejs-devtools/blob/3b776fc0f71369cda72f314e0ec00f4be359f3c9/packages/shell-chrome/views/panel.edge#L16 and https://github.com/alpine-collective/alpinejs-devtools/blob/3b776fc0f71369cda72f314e0ec00f4be359f3c9/packages/shell-chrome/views/panel.edge#L18)

Should keep the selectors simpler and it matches our use-case (we're selecting using the testid's for dev/testing purposes only)

Reply:
Updated now if you want to check it

Reply:
Looks good to me